.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,67 +1,72 @@
-[Short description explaining the high-level reason for the pull request]
+_[Enter an explanation of what the pull request does and why.]_
 
-## Additions
 
--
-
-## Removals
-
--
+---
 
-## Changes
+_[Feel free to delete any checkboxes that are not applicable to this PR.]_
```

Comment:
Looks like this line is in here twice. Did you mean to have it only on line 24?

Reply:
I meant to change the wording here to "sections" instead of "checkboxes".

Reply:
Bumping this. This sentence sounds good if `sections` -> `checkboxes`.

Reply:
Fixed in d3131aca2f5fde0f48ee0a4d3c8f410a120f8acb.